include/seqan3/alignment/pairwise/align_pairwise.hpp

```diff
@@ -134,23 +168,34 @@ constexpr auto align_pairwise(sequence_t && seq, alignment_config_t const & conf
                   std::ranges::ViewableRange<std::tuple_element_t<1, std::remove_reference_t<sequence_t>>>,
                   "Alignment configuration error: The tuple elements must model std::ranges::ViewableRange.");
 
-    return align_pairwise(std::view::single(std::forward<sequence_t>(seq)), config);
+    return align_pairwise(seqan3::seq, std::forward<sequence_t>(seq), config);
```

Comment:
no view-wrapping anymore?

Reply:
This function delegates to the exec aware interface which does now the view wrapping